bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/browser/DefaultWorkbenchBrowserSupport.java

```diff
@@ -53,7 +53,7 @@ IWebBrowser findBrowser(String id) {
 	}
 
 	protected IWebBrowser doCreateBrowser(int style, String browserId, String name, String tooltip)
```

Comment:
Please check the java warning

Reply:
The java warnings are preexisting.

before:
![image (3)](https://github.com/eclipse-platform/eclipse.platform.ui/assets/122080218/f1eade35-ed03-4a89-a527-7365627e6336)

![image (5)](https://github.com/eclipse-platform/eclipse.platform.ui/assets/122080218/dc700e1e-6a63-4212-bef9-2767741ff246)


after:
![image (6)](https://github.com/eclipse-platform/eclipse.platform.ui/assets/122080218/40875c5e-ceb0-47a5-a6df-477993134a6d)